iohk-monitoring/iohk-monitoring.cabal

```diff
@@ -126,7 +126,7 @@ library
                        transformers,
                        unordered-containers,
                        vector,
-                       yaml, libyaml < 0.1.1
+                       yaml, libyaml
```

Comment:
Not sure why this upper bound exists. The latest LTS specifies `libyaml == 0.1.1`.

Reply:
The Buildkite Nix build failed on a configuration parsing test case. Maybe this has something to do with it?

Reply:
That upper bound was due to changes between versions `0.1.0.0` and `0.1.1` of `libyaml`.